bugwarrior/docs/services/jira.rst

```diff
@@ -130,6 +130,13 @@ makes sense when Kerberos is not being used (see above).
 This is useful in situations where HTTP-Basic auth is disabled or disallowed
 for some reason.
 
+When using API token
+++++++++++++++++++++
+
+Some hosts only support API tokens to authenticate. If so, ``bugwarrior-pull`` will respond with ``Err: 401 Unauthorized``. Create a token here_. Handle the token like it is a password.
+
+.. _here: https://id.atlassian.com/manage-profile/security/api-tokens
```

Comment:
Would this link work for users of hosted jira instances? Wouldn't they have a different domain?

Reply:
I am not sure, I have no experience with hosted JIRA instances.